runtime/metricsview/ast.go

```diff
@@ -372,6 +372,21 @@ func (a *AST) resolveMeasure(qm Measure, visible bool) (*runtimev1.MetricsViewSp
 		}, nil
 	}
 
+	if qm.Compute.PercentOfTotal != nil {
+		m, err := a.lookupMeasure(qm.Compute.PercentOfTotal.Measure, visible)
```

Comment:
For safety, let's add a check/error that `qm.Compute.PercentOfTotal.Total` is not `nil` at this point